<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>477</width>
    <height>187</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>477</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавить устройство для отслеживания</string>
  </property>
  <widget class="QLineEdit" name="ipEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>32767</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ipLb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IP адресс:</string>
   </property>
  </widget>
  <widget class="QLabel" name="methodLb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Метод отслеживания:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Обычно достаточно метода PING
для отслеживания - находится ли устройство в сети, 
но иногда ICMP протокол может быть заблокирован
правилами фаервола.
В этом случае нужно использовать конкретный
сетевой сервич для отслеживания, например - HTTP порт 80.
ONFIV метод более детально отслеживает работу IP камер. </string>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <item>
    <property name="text">
     <string>PING - Проверка по ICMP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PORT - Проверка порта TCP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ONVIF - Проверка камеры</string>
    </property>
   </item>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>470</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>470</width>
     <height>100</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="port">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">123</string>
    </property>
    <widget class="QLabel" name="portLb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Порт:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="portEdit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string>99999</string>
     </property>
     <property name="text">
      <string>80</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="auth">
    <widget class="QLabel" name="luserLb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Логин:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>admin</string>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="cursorPosition">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="passLb">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Пароль:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>admin</string>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="cursorPosition">
      <number>1</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="watchStateLb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Следить за устройством:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="watchStateCb">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Пока находится ONLINE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Пока находится OFFLINE</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ipEdit</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>watchStateCb</tabstop>
  <tabstop>portEdit</tabstop>
  <tabstop>loginEdit</tabstop>
  <tabstop>passEdit</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
